free-3.md

```diff
@@ -10,41 +10,108 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My idea will help people organize the tasks they need to do and create a 
+concrete plan of action that they can take. It helps people who struggle with 
+productivity but might be overwhelmed on how to overcome it. There are many
+productivity apps and task managers, but this language would allow someone
+to input their tasks and randomly schedule the tasks into a daily calendar
```

Comment:
Why random? I know that personally I do much better if I have a definitive schedule, and I always do thing X at time Y.